### ISSUE TITLE
No such file or directory

### ISSUE THREAD

**Issue by Mahen27** [Genymobile/scrcpy#1255]
When i am running scrccpy, I am getting the below error, can someone help me on this 

m-c02yh1hajg5h:scrcpy m0c04li$ scrcpy
-bash: /usr/local/bin/scrcpy: No such file or directory

**Comment by rom1v** [comment]
How did you install it?

**Comment by Mahen27** [comment]
1st - brew install scrcpy

Then 

brew cask install android-platform-tools

**Comment by Mahen27** [comment]
@rom1v : any update on this?

**Comment by rom1v** [comment]
Hmm. Could you locate `scrcpy` on your system? Is it in `/usr/bin/scrcpy` for example?

**Comment by artemis-prime** [comment]
Still seeing this.

I installed on Linux using apt, then uninstalled using apt.  Then installed a more recent version using snap.  Which finds it at /snap/bin/scrcpy (and /snap/bin is in my path).  But just running scrcpy yeilds: bash: /usr/bin/scrcpy: No such file or directory

**Comment by artemis-prime** [comment]
Actually this appears to be some sort of file system caching issue.  I suspected as much, since there was no file at `/usr/bin/scrcpy`, so I restarted my system.  Problem gone.  Maybe leave this for others to no waste time over complicating it! ;)

**Comment by syncMutex** [comment]
yes as @artemis-prime mentioned, I too installed by cloning the repo and running `./install-release.sh` and it showed that `/usr/bin/scrcpy` not found even though the binary was in the correct directory. So I just closed and opened the terminal and it worked :smile:.
